mtags/src/main/scala-3/scala/meta/internal/pc/completions/NamedArgCompletions.scala

```diff
@@ -237,15 +237,22 @@ object NamedArgCompletions:
         .getOrElse(baseArgs)
         .filterNot(isUselessLiteral)
 
+      def isDefaultParam(t: Tree): Boolean = t match
```

Comment:
Could be wrong, but looking at the implementation maybe this could be marked as a `@tailrec` function? :slightly_smiling_face:

Reply:
You're right, thanks!